packages/react-core/src/components/Form/Form.tsx

```diff
@@ -9,24 +9,34 @@ export interface FormProps extends React.HTMLProps<HTMLFormElement> {
   className?: string;
   /** Sets the Form to horizontal. */
   isHorizontal?: boolean;
-  /** Flag to limit the max-width to 500px. */
+  /** Limits the max-width of the form. */
   isWidthLimited?: boolean;
+  /** Sets a custom max-width when using isWidthLimited. */
```

Comment:
```suggestion
  /** Sets a custom max-width when using the is width limited prop. */
```

I don't love this phrasing but can't think of anything better at the moment. The goal was just to get away from having camel case `isWidthLimited` in the prop description, but just getting rid of the camel case makes the sentence read weirdly.

Related page from the PF guidelines: https://www.patternfly.org/v4/ux-writing/capitalization/#capitalization-across-patternfly. Props aren't explicitly mentioned though, so maybe this is fine. @tlabaj thoughts?

Reply:
I think that this is a special case and we should leave the prop name other wise it does not make sense like you said @wise-king-sullyman